src/components/Menu/Menu/hooks/useMenuKeyboardNavigation.jsx

```diff
@@ -22,7 +22,6 @@ export default function useMenuKeyboardNavigation(
   setActiveItemIndex,
   isVisible,
   ref,
-  resetOpenSubMenuIndex,
```

Comment:
you can't remove it since it is not an object (it is in method params)
@sahariko

Reply:
I'm not sure I understood you - there's only one place that references this function (in [Menu.jsx](https://github.com/mondaycom/monday-ui-react-core/pull/492/files#diff-bd738b83ed2d4d19d546bde5ecfd6ec076a1d38691922f6098b3b79bdb8931deL86)), and I removed it too in this PR.. where do you think this may still be used?

Reply:
This would be considered a breaking change, resulting in a major version bump of this package.

I think that we can create a milestone and aggregate all of these breaking changes, but for now let's hold back on this.

Reply:
I'm not 100% (I don't think we can be 100% sure). 
It is however a highly internal hook for the menu component, which isn't exposed directly from the package. If someone external imports this hook directly (e.g. `require("dist/src/components.....")`, should we avoid changing it?

I have no problem with reverting this change, but I'm pretty sure I'm missing something :)

Reply:
Hmmm good point, I think we can make this change then 🙂